lib/config-migration.js

```diff
@@ -386,6 +382,11 @@ function migrateComplianceTriggers (config) {
     getConfigFields(codes, config).global.map(f => [f.code, f.value])
   )
 
+  if (!_.isNumber(global.smsVerificationThreshold))
+    return {
+      triggers: []
+    }
+
```

Comment:
I don't think we want this. There can be other requirements active without a sms threshold. What I meant on the my comment was to test both *VerificationActive and *Threshold for every IF of the triggers creation logic.